docs/snapshots.md

```diff
@@ -1,44 +1,47 @@
 
-## Creating and sharing private team snapshots
+Snapshots are a great tool for debugging **remotely** and **asynchronously**--two necessities in the current age of work-from-home. Snapshots eliminate the need for Slack back-and-forth when diagnosing a problem; instead of guessing what error message to copy and paste, a developer can just send their entire Tilt state. Likewise, the frozen-in-time nature of Snapshots makes async debugging a breeze: if a dev encounters strange behavior and the right person to ask isn't online, it's easy to take a snapshot, send over the link, and go back to writing code. 
 
-[Ensure your project is associated with a Tilt Cloud team using `set_team` in your Tiltfile.](teams.html) Make sure you are either a [`Member` or `Owner`](teams.html#add-users-to-your-team) of the team.
-
-Inside Tilt, click the `Create a snapshot` button at the top. If you haven't yet connected Tilt to Tilt Cloud, you'll be prompted to do so (using your GitHub account to sign in). Click `GET LINK` to create the snapshot. Click `OPEN` and Tilt will open the snapshot in Tilt Cloud in a new browser tab. Copy the URL of the snapshot and share it with other team members (via Slack or email, for example). The snapshot is only accessible to Tilt Cloud signed-in users who are a member (or owner) of the team
-
-**Important**: If the Tiltfile errors on its first execution when you run `tilt up`, and [`set_team`](https://docs.tilt.dev/api.html#api.set_team) has not been called yet, Tilt will not understand `set_team`. If you then take a snapshot, it will be a _public_ snapshot. (See below for public snapshots.) To work around this problem, make sure to put `set_team` as the first line in your Tiltfile. Tilt will then successfully parse `set_team` and start using the team id in it. Create a snapshot as normal and it will be a private team snapshot.
+## Creating and Sharing Snapshots
+Inside Tilt, click the `Create a snapshot` button at the top. If you haven't yet connected Tilt to Tilt Cloud, you'll be prompted to do so (using your GitHub account to sign in). Click `GET LINK` to create the snapshot. Click `OPEN` and Tilt will open the snapshot in Tilt Cloud in a new browser tab. Copy the URL of the snapshot and share at will.
 
+// TOOD: maybe a gif here instead?
 ![create-snapshot](assets/img/create-snapshot.png)
 
-## Viewing all private team snapshots
+Visit [cloud.tilt.dev/snapshots](https://cloud.tilt.dev/snapshots) for a list of all active snapshots you have created.
 
-Click the `Create a snapshot` button in Tilt. In the modal, click `View snapshots from your team`. to see a list of all private team snapshots, for that team, in Tilt Cloud.
+### Private Snapshots
```

Comment:
```suggestion
### Private team snapshots
```

I think we should emphasize "team" here somehow. I want to make it explicit that there's only two types of snapshots:

- Team snapshots, which are private.
- Non-team snapshots, which are public.

I don't want a reader having to wonder, can I have a public team snapshot?

Reply:
Hmm I think that the phrasing "private team snapshots" does itself suggest the existence of "public team snapshots". What about:
> Private Snapshots (aka Team Snapshots)
?

Reply:
Yep I like your latest suggestion